src/test/ui/fn/fn-fat-arrow-return.rs

```diff
@@ -0,0 +1,4 @@
+fn g() => usize { 0 }
+//~^ ERROR return types are denoted using `->`
```

Comment:
Could you also test `cargo fix`?
```suggestion
// run-rustfix
fn g() => usize { 0 }
//~^ ERROR return types are denoted using `->`
```
and then run `x.py test src/test/ui --bless --test-args fat-arrow-return`